From ca72b7dcf6d5cca3649fdb5d30bab580dac930b2 Mon Sep 17 00:00:00 2001
From: Le Hoang Minh Thuan <thuanlhm@fsoft.com.vn>
Date: Tue, 5 Nov 2019 15:17:39 +0700
Subject: [PATCH 631/631] ARM: shmobile: r8a77470 use setup rcar gen2 and
 platsmp apmu

Switch to use setup-rcar-gen2 and platsmp-apmu from community.

Signed-off-by: Danh Duong <danhdc1@fsoft.com.vn>
Signed-off-by: Thao M. Hoang <thaohm2@fsoft.com.vn>
Signed-off-by: Le Hoang Minh Thuan <thuanlhm@fsoft.com.vn>
---
 arch/arm/mach-shmobile/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-shmobile/Makefile b/arch/arm/mach-shmobile/Makefile
index 788b48c..96ee2f6 100644
--- a/arch/arm/mach-shmobile/Makefile
+++ b/arch/arm/mach-shmobile/Makefile
@@ -15,6 +15,7 @@ obj-$(CONFIG_ARCH_R8A7790)	+= setup-r8a7790.o
 obj-$(CONFIG_ARCH_R8A7790)	+= setup-r8a7742.o
 obj-$(CONFIG_ARCH_R8A7791)	+= setup-r8a7791.o
 obj-$(CONFIG_ARCH_R8A7745)      += setup-r8a7745.o sgx-r8a7745-quirk.o
+obj-$(CONFIG_ARCH_R8A77470)     += setup-rcar-gen2.o
 obj-$(CONFIG_ARCH_R8A7744)	+= setup-r8a7744.o
 obj-$(CONFIG_ARCH_EMEV2)	+= setup-emev2.o
 obj-$(CONFIG_ARCH_R7S72100)	+= setup-r7s72100.o
@@ -37,6 +38,7 @@ smp-$(CONFIG_ARCH_R8A7779)	+= smp-r8a7779.o headsmp-scu.o platsmp-scu.o
 smp-$(CONFIG_ARCH_R8A7790)	+= smp-r8a7790.o
 smp-$(CONFIG_ARCH_R8A7790)	+= smp-r8a7742.o
 smp-$(CONFIG_ARCH_R8A7791)	+= smp-r8a7791.o
+smp-$(CONFIG_ARCH_R8A77470)     += platsmp-apmu.o $(cpu-y)
 smp-$(CONFIG_ARCH_R8A7744)	+= smp-r8a7744.o
 smp-$(CONFIG_ARCH_EMEV2)	+= smp-emev2.o headsmp-scu.o platsmp-scu.o
 
-- 
2.7.4

